### ISSUE TITLE
Can't upgrade to v.2.6

### ISSUE THREAD

**Issue by mvpopuk** [statamic-rad-pack/mailchimp#74]
I'm on Statamic `v.3.2.36` and `silentz/mailchimp v.2.5.7`. When I try to upgrade to `v.2.6` I get the following:

```  
 Problem 1
    - silentz/mailchimp[v2.5.8, ..., v2.6] require statamic/cms 3.2.32 -> found statamic/cms[v3.2.32] but it conflicts with your root composer.json require (3.2.36).
    - Root composer.json requires silentz/mailchimp ^2.5.8 -> satisfiable by silentz/mailchimp[v2.5.8, v2.6].
```

In my `composer.json` I have:

```
"silentz/mailchimp": "^2.5", (at the moment)
"statamic/cms": "3.2.36",
```

Not sure what I'm doing wrong.

**Comment by edalzell** [comment]
like it says, Mailchimp, at the moment, only supports Statamic 3.2.32. Set you composer version to that.

Once this [PR](https://github.com/statamic/cms/pull/5424) gets merged and released, I'll update the version constraint.

**Comment by edalzell** [comment]
Released 2.6.1 that requires Statamic 3.2.37, which fixes the issue
